### ISSUE TITLE
Cannot find a module definition

### ISSUE THREAD

**Issue by ktrzos** [reasonml-editor/reasonml-idea-plugin#236]
I'm using DateFns library and when I'm clicking combination of `ctrl + B`, on
`open ReasonDateFns;` (I want to go to the definition of this module), this warning appears:
 
![image](https://user-images.githubusercontent.com/14212721/80376883-c2a29f00-889a-11ea-9eae-3996e8b1835c.png)

This library is installed, and everything is compiling, so there must be something with the plugin.

**Comment by hartmut27** [comment]
Omit the `Reason...` prefix please in your `open ...` statement.
The lookup with `ctrl + b` is then working correctly with `open DateFns`.
Is this what you're searching for?
![image](https://user-images.githubusercontent.com/752620/80383262-a6573000-88a3-11ea-9755-838560d82436.png)

**Comment by hartmut27** [comment]
This could occur possibly because of `reason-date-fns` is the internal package name:

`However, this means that you can't use a camelCased names such as MyProject, since package.json and npm forbid you to do so (some file systems are case-insensitive). To have the namespace/module as MyProject, write "name": "my-project". Bsb will turn that into the camelCased name correctly.` [bucklescript configuration](https://bucklescript.github.io/docs/en/build-configuration.html#name-namespace)

?

**Comment by hartmut27** [comment]
In a different scenario I receive a similar message. The message appears and the lookup `ctrl + b` works though.
![image](https://user-images.githubusercontent.com/752620/80398734-65b5e180-88b8-11ea-9f50-6ea7c5a4e0f1.png)


`The module or file Alcotest can't be found. - If it's a third-party dependency: - Did you list it in bsconfig.json? - Did you run `bsb` instead of `bsb -make-world` (latter builds third-parties)? - Did you include the file's directory in bsconfig.json?`

**Comment by hartmut27** [comment]
Duplicate of #213

**Comment by ktrzos** [comment]
@hartmut27 
Ok, so do you have any idea how to manage with this problem? Is it a matter of wrong configuration of `bsconfig.json`?

As for current status of this file in my project, I've got `reason-apollo-hooks` in here:

```
  "bs-dependencies": [
    ...
    "reason-apollo-hooks",
    ...
  ],
```

**Comment by hartmut27** [comment]
@ktrzos My guess:  This is a _more common_ plugin problem _not_ related to wrong configuration setting. As a workaround I would propose to use the "Find In Path..." function on IntelliJ meanwhile  to get to the libraries sourcecodes.
Please check that node_modules folder is _not excluded_ ("Mark directory as..." function of IntelliJ).
( On top of this I find it very useful to have the OCaml Standard Libraries also in the search space. I've added `/home/username12345/.opam/4.09.1/lib` to IntellIJs SDK sourcepaths. )
![image](https://user-images.githubusercontent.com/752620/80454584-96cefa00-892a-11ea-893e-8ad55ce83306.png)`.

**Comment by giraud** [comment]
DateFns has a `namespace: true` in its bsconfig.json. It seems to be the problem

**Comment by giraud** [comment]
I found a way to index module with a virtual namespace. That'll help in a lot in module resoltuion

**Comment by giraud** [comment]
It should be fixed in 0.89. If not, try to invalidate caches (but it should not be needed)

**Comment by ktrzos** [comment]
Great! Thanks :).
